app/views/roles/show.html.erb

```diff
@@ -7,7 +7,7 @@
 <%= render partial: "current_holder", locals: { role: role } %>
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-two-thirds" dir="<%= page_text_direction %>" lang="<%= role.locale %>">
```

Comment:
Same here - this shouldn't be `rtl` as it contains a left-to-right language and the lang is always added regardless of the default language.